src/__tests__/utils.spec.ts

```diff
@@ -165,6 +165,13 @@ describe('testing utils', () => {
 
       expect(codeRef).toBe('');
     });
+
+    test('should return correct code reference for test title (including project name)', () => {
+      const expectedCodeRef = 'Google Chrome tests/tests/example.js/rootDescribe';
+      const codeRef = getCodeRef(mockedTest, 'rootDescribe');
+
+      expect(codeRef).toBe(expectedCodeRef);
+    });
```

Comment:
This case already covered by test at line [149](https://github.com/reportportal/agent-js-playwright/pull/54/files#diff-ac6041778b61d222775fbee7a48d2c432b352b13be942afa22dfbf99089ff965R149), so please remove it.